cpp/src/slice2java/Gen.cpp

```diff
@@ -3624,8 +3624,7 @@ Slice::Gen::TypesVisitor::visitInterfaceDefEnd(const InterfaceDefPtr& p)
             "@param communicator The communicator of the new proxy.\n"
             "@param proxyString The string representation of the proxy.\n"
             "@return The new proxy.");
-    out << nl << "public static " << prxName
-        << " createProxy(com.zeroc.Ice.Communicator communicator, String proxyString)";
+    out << nl << "static " << prxName << " createProxy(com.zeroc.Ice.Communicator communicator, String proxyString)";
```

Comment:
is this because they are implicitly public?

Reply:
Yes, and the preferred style in Java is to omit public for interface methods.